uwuno/src/test/java/com/learning/uwuno/integration/tests/basicPlayerTest.java

```diff
@@ -193,7 +193,7 @@ public void putValidPlayerName200() throws FileNotFoundException {
     public void putInvalidPlayerName400() throws FileNotFoundException {
         // Player
         String playerFilePath = JSON_REQUESTS_PATH + "/postPutPlayer.json";
-        String playerInputName = "player_put_old_name_valid";;
+        String playerInputName = "player_put_old_name_valid";
```

Comment:
Oops didn't see this before, if this string is used in multiple places should move to the top and only create one string?